tests/__fixtures__/Rules/newline_before_scenario/fixture.js

```diff
@@ -0,0 +1,559 @@
+            `Feature: a feature file
+  Scenario Outline: a scenario outline`,
+            [generateProblem({ line: 2, column: 3 }, 0)],
+        ],
+        [
+            "without Rule: Background - Scenario Outline",
+            `Feature: a feature file
+  Background: a background
+  Scenario Outline: a scenario`,
+            [generateProblem({ line: 3, column: 3 }, 0)],
+        ],
+        [
+            "without Rule: multiple scenarios",
+            `Feature: a feature file
+  Background: a background
+
+
+  Scenario: a scenario
+
+
+
+  Scenario Outline: a scenario outline`,
+            [
+                generateProblem({ line: 5, column: 3 }, 2),
+                generateProblem({ line: 9, column: 3 }, 3),
+            ],
+        ],
+        [
+            "without Rule: description - Scenario",
+            `Feature: a feature file
+  Some feature description
+  is here
+  Scenario: a scenario`,
+            [generateProblem({ line: 4, column: 3 }, 0)],
+        ],
+        [
+            "without Rule: Scenario tags",
+            `Feature: a feature file
+
+
+  @tag1 @tag2
+
+  Scenario: a scenario`,
+            [generateProblem({ line: 6, column: 3 }, 3)],
+        ],
+        [
+            "without Rule: Scenario multiline tags",
+            `Feature: a feature file
+
+  @tag1
+
+  @tag2
+  Scenario Outline: a scenario outline`,
+            [generateProblem({ line: 6, column: 3 }, 2)],
+        ],
+        [
+            "without Rule: Scenario comments",
+            `Feature: a feature file
+  #comment line
+
+
+  Scenario: a scenario`,
+            [generateProblem({ line: 5, column: 3 }, 2)],
+        ],
+        [
+            "without Rule: description - Scenario tags and comments",
+            `Feature: a feature file
+  feature description here
+
+  @tag1
+
+  #comment line
+
+  @tag2
+  Scenario: a scenario`,
+            [generateProblem({ line: 9, column: 3 }, 3)],
+        ],
+        [
+            "without Rule: Background - Scenario tags and comments",
+            `Feature: a feature file
+  feature description here
+
+  Background: a background
+    describe a background
+  @tag1
+  #comment line
+  @tag2
+  Scenario: a scenario`,
+            [generateProblem({ line: 9, column: 3 }, 0)],
+        ],
+        [
+            "Step datatable",
+            `Feature: a feature file
+  Background: a background
+    Given a step
+      | data1 |
+      | data2 |
+  Scenario: a scenario`,
+            [generateProblem({ line: 6, column: 3 }, 0)],
+        ],
+        [
+            "Step docstring",
+            `Feature: a feature file
+  Background: a background
+    Given a step
+      """
+      multiline
+      content
+      """
+
+
+  Scenario: a scenario`,
+            [generateProblem({ line: 10, column: 3 }, 2)],
+        ],
+        [
+            "Examples table",
+            `Feature: a feature file
+
+  Scenario: a scenario
+    Given a step
+  Examples:
+      | row1 |
+      | row2 |
```

Comment:
This also looks like a Scenario Outline - maybe it does not matter in the tests for this rule.